components/layout/layout_task.rs

```diff
@@ -127,6 +127,9 @@ pub struct LayoutTaskData {
     /// A queued response for the content boxes of a node.
     pub content_boxes_response: Vec<Rect<Au>>,
 
+    /// A queued response for the offset parent/rect of a node.
+    pub offset_parent_response: OffsetParentResponse,
+
```

Comment:
@jdm opened https://github.com/servo/servo/issues/6787 - perhaps we can do this as a follow up?